src/View/Admin/Faq/adminAdd.html.twig

```diff
@@ -0,0 +1,27 @@
+{% extends 'layout.html.twig' %}
+{% block title %}
+	{{parent()}}
+	|Faq|Ajout
+{% endblock %}
+
+{% block content %}
+
+	<main class="container">
+		<h1>Ajouter FAQ</h1>
+		<ul>
+			{% for error in errors %}
+				<li>{{ error }}</li>
+			{% endfor %}
+		</ul>
+		<form action="" method="post">
+			<label for="question">Question:
+			</label>
+			<input type="text" value="{{ item.question }}" name="question">
```

Comment:
en bdd tu as mis un TEXT pour les 2 non? donc soit tu mets des VARCHAR en bdd et des input type="text";
soit tu laisse des TEXT mais ce sera des textarea dans le form

Reply:
et si c'est not null, mets un required